io/net/virt-net/lpm.py.data/lpm.yaml

```diff
@@ -1,3 +1,5 @@
 hmc_pwd:
 hmc_username:
 remote_server:
+lpar_ip:
```

Comment:
I would create a new file called lpm_from_peer.yaml so it won't affect the FVT team's current configuration. Additionally, create a new .cfg file in the avocado-fvt-wrapper repo called io_vnic_lpm_from_peer.cfg with the necessary changes(change the yaml file). 
I think you're going to have to do the same to the lpm_5_iterations.yaml file as well.